queue.tf

```diff
@@ -0,0 +1,13 @@
+module "queue-namespace" {
+  source              = "git@github.com:hmcts/terraform-module-servicebus-namespace.git"
+  name                = "${var.product}-servicebus-${var.env}"
+  location            = "${var.location}"
+  resource_group_name = "${azurerm_resource_group.rg.name}"
+}
+
+module "queue" {
+  source              = "git@github.com:hmcts/terraform-module-servicebus-queue.git"
+  name                = "envelopes"
```

Comment:
Queue holds just envelopes id's and not the envelopes so may just name it as `envelope-id-queue` or something similar?

Reply:
happy to change, @lgonczar  any reason why it was just `envelopes` before?